### DISCUSSION TITLE
vant/cli2.0封装组件库后导入到vue2项目中使用，toast被popup覆盖住了

### DISCUSSION THREAD

**Discussion by wanshneyin** [youzan/vant#11875] [Q&A]
背景：vant/cli2.x二次封装vant组件库在vue2项目中使用（vue2项目cdn全局引入了vant）

问题描述：在vant/cli2.x封装的组件中按需引入vant组件库的组件，当我publish后，再将我二次封装的包引入到vue2项目中使用时，项目中之前使用的vant组件的样式发生了改变（比如：toast被popup覆盖住了，因为toast的z-index从2000开始了，而popup是2002）

报错：
![image](https://github.com/youzan/vant/assets/67151403/9fb1ea64-b380-4013-973c-fba866f86fa7)

**Comment by chenjiahan** [comment]
你在构建二次封装的组件库时，可以把 Vant 的代码和样式 externals 掉，由业务项目来引入，业务项目可以自行决定是通过 CDN 引入还是 npm 引入。
